scripts/simplify.py

```diff
@@ -0,0 +1,249 @@
+	for image in images:
+		new_image = xml.createElement('image')
+
+		# Recreate each allowed attribute in the newly created element
+		for allowed_attr in image_allowed_attrs:
+			# Check if field has a value
+			if not image.getAttribute(allowed_attr):
+				continue
+
+			new_image.setAttribute(allowed_attr, image.getAttribute(allowed_attr))
+
+		# Assign default image style
+		new_image.setAttribute('style', dict_to_style(default_image_style))
+
+		# Add UUID if needed
+		if not new_image.getAttribute('pcbre:uuid'):
+			new_image.setAttribute('pcbre:uuid', str(uuid.uuid4()))
+
+		# Sort attributes
+		sort_element_attr(new_image)
+
+		new_layer.appendChild(new_image)
+
+	# Sort elements
+	new_layer.childNodes.sort(key=lambda x: x.getAttribute('pcbre:uuid'))
+
+def process_zones_layer(xml, layer, new_layer):
+	zones = layer.getElementsByTagName('path')
+
+	trace_allowed_attrs = ['d', 'transform', 'inkscape:connector-curvature', 'inkscape:label', 'pcbre:uuid']
+	for zone in zones:
+		new_zone = xml.createElement('path')
+
+		# Recreate each allowed attribute in the newly created element
+		for allowed_attr in trace_allowed_attrs:
+			# Check if field has a value
+			if not zone.getAttribute(allowed_attr):
+				continue
+
+			new_zone.setAttribute(allowed_attr, zone.getAttribute(allowed_attr))
+
+		# TODO: Remove transforms
+
+		# Assign default zone style
+		new_zone.setAttribute('style', dict_to_style(default_zone_style))
+
+		# Assign default zone label
+		if not new_zone.getAttribute('inkscape:label'):
+			new_zone.setAttribute('inkscape:label', default_zone_label)
+
+		# Add UUID if needed
+		if not new_zone.getAttribute('pcbre:uuid'):
+			new_zone.setAttribute('pcbre:uuid', str(uuid.uuid4()))
+
+		# Sort attributes
+		sort_element_attr(new_zone)
+
+		new_layer.appendChild(new_zone)
+
+	# Sort elements
+	new_layer.childNodes.sort(key=lambda x: x.getAttribute('pcbre:uuid'))
+
+def process_traces_layer(xml, layer, new_layer):
+	traces = layer.getElementsByTagName('path')
+
+	trace_allowed_attrs = ['d', 'inkscape:connector-curvature', 'pcbre:uuid']
+	for trace in traces:
+		new_trace = xml.createElement('path')
+
+		# Recreate each allowed attribute in the newly created element
+		for allowed_attr in trace_allowed_attrs:
+			# Check if field has a value
+			if not trace.getAttribute(allowed_attr):
+				continue
+
+			new_trace.setAttribute(allowed_attr, trace.getAttribute(allowed_attr))
+
+		# Assign default trace style
+		new_trace.setAttribute('style', dict_to_style(default_trace_style))
+
+		# Add UUID if needed
+		if not new_trace.getAttribute('pcbre:uuid'):
+			new_trace.setAttribute('pcbre:uuid', str(uuid.uuid4()))
+
+		# Sort attributes
+		sort_element_attr(new_trace)
+
+		new_layer.appendChild(new_trace)
+
+	# Sort elements
+	new_layer.childNodes.sort(key=lambda x: x.getAttribute('pcbre:uuid'))
+
+def main(argv):
+    # Load and parse SVG
+	input_xml = minidom.parse(argv[0])
```

Comment:
Consider using `argparse`, or use `sys.argv` directly for readability.
I think `argv[0]` looks like a bug (implies `sys.argv[0]`), but it's correct because it's actually `sys.argv[1]` in this case.

Reply:
This was mostly a place holder until a better method of handling the command arguments was implemented. I see the need for additional arguments in the future and I would rather we go ahead and choose a library to handle that instead. At this time I don't have any recommendations on what would be the best way to do this.

Reply:
https://docs.python.org/3/library/argparse.html is standard in python